apps/omg_watcher/lib/omg_watcher/application.ex

```diff
@@ -13,15 +13,13 @@
 # limitations under the License.
 
 defmodule OMG.Watcher.Application do
```

Comment:
why remove logging and the other things from here? seems like they should stay

Reply:
how exactly would default comments help you? :) these comments come from `mix new`

Reply:
@pdobacz From talking to Ino...

1. `spandex-query-tracer` we don't need because watcher does not have postgres db anymore.
2. Erlang cookies are being managed by releases now, i.e. `-setcookie ${ERLANG_COOKIE}` in `rel/vm.args`.

Reply:
Ah but I see about `_ = Logger.info("Starting #{inspect(__MODULE__)}")`. Adding back now.

Reply:
ah wait. I got confused apparently. I though we removed ```
  # See https://hexdocs.pm/elixir/Application.html
  # for more information on OTP Applications
```
:D